### ISSUE TITLE
LXC support

### ISSUE THREAD

**Issue by lsmith77** [geerlingguy/drupal-vm#1911]
### Issue Type

Feature Idea

### Your Environment

```
* Vagrant 2.2.2
* VirtualBox 5.2.18_Ubuntur123745
```

### Your OS

**Linux** (Ubuntu 18.04)

### Summary

We didn't manage to get LXC working with the boxes listed here:
https://github.com/geerlingguy/drupal-vm/blob/master/docs/other/vagrant-lxc.md#modify-your-configyml

Some of dissapeared others are not working.

We are currently using `vagrant_box: geerlingguy/ubuntu1604`. Any plans to make this box work wiht LXC or any other suggestions?

**Comment by geerlingguy** [comment]
As I don't currently use LXC directly I'm not inclined to work on it; but please see this older issue where many different potential paths towards doing that successfully were explored: https://github.com/geerlingguy/drupal-vm/issues/649

**Comment by lsmith77** [comment]
@geerlingguy ok .. maybe you can answer me one more question. I am still a Drupal n00b and therefore also Drupal VM n00b.

from my understanding Drupal VM supports different vagrant boxes. here are at Liip we have one office using the standard VM with Drupal VM and the other is using https://github.com/liip/drifter which is LXC compatible but there the team does not use Drupal VM.

Now I wonder can you give me some hints about what we would have to do to make Drifter work as Drupal VM base?

**Comment by geerlingguy** [comment]
@lsmith77 - It looks like Drifter might be synonymous to Drupal VM. Drupal VM is basically a set of Ansible roles and playbooks which use Vagrant to manage a VM.

Using VirtualBox is a default implementation detail; in the past I used to also maintain a VMware Fusion base box and support Parallels, but since that required me to pay a few hundred bucks a year to maintain licenses I gave up. And since I don't run a Linux machine as a daily driver I don't maintain an LXC base box.

However, some (many) people still use Drupal VM successfully with Parallels and Fusion, and some have found ways to make it work with LXC and HyperV even... I just don't officially support less common (and sometimes non-free) configurations.

But back to your point, if there's anything Drupal VM can change to make it easier to use LXC instead of VirtualBox on Linux, which doesn't add additional maintenance burden, I'd be more than happy to merge it in. I will not be working on that myself though.
